subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/serialization/codecs/TaskNodeCodec.kt

```diff
@@ -170,7 +170,7 @@ class TaskNodeCodec(
 
     private
     suspend fun WriteContext.writeRegisteredServicesOf(task: TaskInternal) {
-        writeCollection(task.requiredServices)
+        writeCollection(task.requiredServices.requiredServices)
```

Comment:
I wish I had a suggestion for a way to avoid the pleonasm here.

Reply:
It is a bit odd, renaming one or the other would minimize it, but at the same time didn't bother me enough to want to change what otherwise seem decent names.